sympy/algebras/quaternion.py

```diff
@@ -9,6 +9,7 @@
 from sympy import Matrix
 from sympy import sympify
 from sympy.core.expr import Expr
+from sympy.core.evalf import prec_to_dps
```

Comment:
Imports are generally sorted alphabetically.

Reply:
@namannimmo10 i don't understand , should that import be included at the top?